pyaerocom/aeroval/experiment_output.py

```diff
@@ -793,61 +809,3 @@ def _sort_menu_entries(self, avail: dict) -> dict:
                     models_sorted = sort_dict_by_name(models, pref_list=model_order)
                     new_sorted[var]["obs"][obs_name][vert_code] = models_sorted
         return new_sorted
-
-    def _add_entry_experiments_json(self, exp_id, data) -> None:
-        fp = self.experiments_file
-        current = read_json(fp)
-        current[exp_id] = data
-        write_json(
-            current,
-            self.experiments_file,
-            indent=4,
-        )
-
-    def _del_entry_experiments_json(self, exp_id) -> None:
-        """
-        Remove an entry from experiments.json
-
-        Parameters
-        ----------
-        exp_id : str
-            name of experiment
-
-        Returns
-        -------
-        None
-
-        """
-        current = read_json(self.experiments_file)
-        try:
-            del current[exp_id]
-        except KeyError:
-            logger.warning(f"no such experiment registered: {exp_id}")
-        write_json(
-            current,
-            self.experiments_file,
-            indent=4,
-        )
-
-    def reorder_experiments(self, exp_order=None) -> None:
```

Comment:
This functionality is only called from tests within pyaerocom. If it is required externally, let me know.